osgp/platform/osgp-logging/src/main/resources/db/migration/V20201224160000000__Add_upper_device_log_item_index.sql

```diff
@@ -0,0 +1,3 @@
+DROP INDEX IF EXISTS device_log_item_device_identification_upper;
+CREATE INDEX IF NOT EXISTS device_log_item_device_identification_upper_modification_time ON device_log_item(UPPER(device_identification), modification_time);
```

Comment:
device_log_item_ix_upper_device_identification_modification_time fits better with our code conventions, which refer to https://launchbylunch.com/posts/2014/Feb/16/sql-naming-conventions/.